index.js

```diff
@@ -11,26 +11,58 @@ const nextButton = document.querySelector('[data-js="button-next"]');
 const pagination = document.querySelector('[data-js="pagination"]');
 
 // States
-const maxPage = 1;
-const page = 1;
+let maxPage = 1; // changed to let
+let page = 1;
 const searchQuery = "";
 
-async function fetchCharacters() {
-  const response = await fetch("https://rickandmortyapi.com/api/character");
-  const data = await response.json();
-
-  return data.results;
+async function fetchCharacters(page) {
+  try {
+    const response = await fetch(
+      `https://rickandmortyapi.com/api/character?page=${page}` // creation of templateString
+    );
+    if (!response.ok) {
+      throw new Error(`status:${response.status}`);
+    }
+    const data = await response.json();
+    maxPage = data.info.pages; //
+    console.log(data);
+    return data.results;
+  } catch (error) {
+    console.log("Error!!!");
```

Comment:
Since you are creating an explicit error message in line 24 you can use it here:
```console.log(`Error, status: ${error.message}`)```